internal/config/state.go

```diff
@@ -218,3 +220,47 @@ func isValidImage(image string) bool {
 
 	return ref != "" && tag != ""
 }
+
+// ReadResponse reads the library state from configure-response.json.
+//
+// The response file is removed afterwards.
+func ReadResponse(contentLoader func(data []byte, state *LibraryState) error, jsonFilePath string) (*LibraryState, error) {
```

Comment:
`ReadResponse` and `WriteLibrarianState` are more suited in librarian/state.go, where load related functions are already located.
followup to https://github.com/googleapis/librarian/pull/989#discussion_r2242731498